sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -3575,11 +3573,22 @@ private <DestinationT> WriteResult continueExpandTyped(
             !getPropagateSuccessfulStorageApiWrites(),
             "withPropagateSuccessfulStorageApiWrites only supported when using storage api writes.");
 
-        // Batch load jobs currently support JSON data insertion only with CSV files
+        // Beam does not yet support Batch load jobs with Avro files
```

Comment:
Remove this comment? I think it's supported for all types now

Reply:
thanks for catching this